### ISSUE TITLE
Error: Private key suspiciously small: < 16 bytes. Aborting!

### ISSUE THREAD

**Issue by milankragujevic** [Paratii-Video/paratii-lib#58]
`yarn test` doesn't pass, shows this: `Error: Private key suspiciously small: < 16 bytes. Aborting!`. Full output: 

```
Milans-MacBook-Air:paratii-lib milankragujevic$ yarn test
yarn run v1.3.2
$ mocha --require babel.js --exit --timeout 10000 test/*.js


  Paratii API:
    ✓ deployContracts should deploy and register all contracts (497ms)

  Paratii API:
    ✓ diagnose() should run without errors (318ms)

  Paratii API:
4
    ✓ example session from ../docs/example-session.md should work (235ms)

  Paratii configuration:
    ✓ paratii.config should return the configuration with default values
    ✓ testnet configuration should be recognized
    ✓ should be possible to create a second Paratii object with the same settings (244ms)
    ✓ should be possible to create a Paratii instance without an address or registryAddress
    ✓ setAccount should set the account
    ✓ paratii.eth.web3 should be available

  paratii.core API: :
    ✓ should be configured
    - core.users.create() should work as expected
    - core.users.get() should work as expected

  paratii.core.vids:
bitswap options:  { maxMessageSize: 32768 }
    ✓ core.vids.create() and get() should work as expected (5024ms)
bitswap options:  { maxMessageSize: 32768 }
    ✓ core.vids.create() should accept ipfsHash as argument (5435ms)
bitswap options:  { maxMessageSize: 32768 }
    ✓ core.vids.update() should work as expected (1567ms)
    - core.vids.delete() should work as expected
    - core.vids.like() should work as expected
    - core.vids.dislike() should work as expected
    - core.vids.view() should work as expected
    - core.vids.buy() should work as expected
    - core.vids.search() should work as expected

  paratii.db API: :
    ✓ should be configured
    - db.vids.get() should work as expected
    - db.users.get() should work as expected
    - db.vids.search() should work as expected
    ✓ should be available as an attribute on Paratii instances

  paratii.eth.events API: :
    ✓ subscription to Tranfer PTI events should work as expected (70ms)
    ✓ subscription to Create Video events should work as expected (45ms)

  paratii.eth API: :
    ✓ getContracts() should return the contracts
    ✓ contracts should have their address set (70ms)
    ✓ getContract() should return the correct contract
    1) balanceOf() should return the right balances
    ✓ transfer ETH should work as expected
    ✓ transfer PTI should work as expected
    ✓ deployContract should throw a sensible error if address is not set

  paratii.eth.users: :
    ✓ users.create(),  .get(), .update() and .delete() should work (83ms)
    - missing or wrong arguments in users.create() should trhow meaningful errors

  paratii.eth.vids:
    ✓ vids.create(),  .get(), .update() and .delete() should work (83ms)
    - vids.create() should throw meaningful errors
    - vids.create() should create a fresh id if no id wat provided

  paratii.eth.wallet: :
    ✓ init account is added to wallet
    ✓ if no account is given, the wallet accounts are empty
    ✓ wallet.create() works (165ms)
    ✓ wallet.create() does not create a new wallet object (156ms)
    ✓ wallet.create() creates a new mnenomic if not mnomic is given (49ms)
    ✓ wallet.encrypt() and decrypt() works (546ms)
    ✓ send() should fail if no wallet is present
    ✓ send() should succeed if a  private key is passed to the constructor (43ms)
    ✓ send() should succeed if a  private key is passed to the constructor (44ms)
    ✓ eth.wallet.isValidMnemonic() should work as expected
    ✓ eth.wallet.newMnemonic() should work as expected
    ✓ eth.wallet.create() should throw if a wallet already has an account
    ✓ eth.wallet.getMnemonic() should work as expected (71ms)

  ParatiiIPFS: :
    ✓ should exist
bitswap options:  { maxMessageSize: 32768 }
    ✓ should create an instance without trouble (2914ms)
bitswap options:  { maxMessageSize: 32768 }
    ✓ should allow for simple add() and get() of files (778ms)
bitswap options:  { maxMessageSize: 32768 }
    ✓ put a JSON object and get it back (3370ms)
bitswap options:  { maxMessageSize: 32768 }
    ✓ should exist and work as an attribute on the Paratii object (870ms)

  ParatiiIPFS: :
{ onDone: [Function: onDone], onError: [Function: onError] }
bitswap options:  { maxMessageSize: 32768 }
Adding  test/data/some-file.txt
Adding QmS8yinWCD1vm7WJx34tg81FpjEXbdYXf3Y5XcCeh29C6K finished
uploader Finished [ undefined ]
    ✓ should allow for file upload (3444ms)
bitswap options:  { maxMessageSize: 32768 }
reading file  genericJson.json
reading file  some-file.txt
this is the hash to return 
    ✓ should add a directory to IPFS (3270ms)
bitswap options:  { maxMessageSize: 32768 }
uploader Finished []
_signalTranscoder Got an empty Array. files:  []
    ✓ addAndTranscode() should work as expected (6580ms)
    - should be able to grab a youtube video and upload it
    - should be able to grab a vimeo video and upload it

  Paratii API:
    ✓ paratii exists..


  47 passing (41s)
  16 pending
  1 failing

  1) paratii.eth API: :
       balanceOf() should return the right balances:
     Error: Private key suspiciously small: < 16 bytes. Aborting!
      at Wallet.create$ (lib/paratii.eth.wallet.js:36:17)
      at tryCatch (node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:62:40)
      at Generator.invoke [as _invoke] (node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:296:22)
      at Generator.prototype.(anonymous function) [as next] (node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:114:21)
      at tryCatch (node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:62:40)
      at invoke (node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:152:20)
      at node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:195:11
      at new Promise (<anonymous>)
      at callInvokeWithMethodAndArg (node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:194:16)
      at AsyncIterator.enqueue (node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:217:13)
      at AsyncIterator.prototype.(anonymous function) [as next] (node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:114:21)
      at Object.runtime.async (node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:241:14)
      at Wallet.create (lib/paratii.eth.wallet.js:7:3)
      at Context._callee5$ (test/paratii.eth.js:76:45)
      at tryCatch (node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:62:40)
      at Generator.invoke [as _invoke] (node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:296:22)
      at Generator.prototype.(anonymous function) [as next] (node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:114:21)
      at tryCatch (node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:62:40)
      at invoke (node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:152:20)
      at node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:195:11
      at new Promise (<anonymous>)
      at callInvokeWithMethodAndArg (node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:194:16)
      at AsyncIterator.enqueue (node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:217:13)
      at AsyncIterator.prototype.(anonymous function) [as next] (node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:114:21)
      at Object.runtime.async (node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:241:14)
      at Context._callee5 (test/paratii.eth.js:72:54)
      at <anonymous>



error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Milans-MacBook-Air:paratii-lib milankragujevic$ 

```

On macOS 10.13 with 8GB of RAM.

**Comment by jellegerbrandy** [comment]
This is an error from a dependency t (bitcore-something) that happens inpredictably.I
It's a bit worrying, because it is part of the wallet-generation code. 
it would be better to fix (or use another lib), but otherwise the workaround could be, in case of error, to just try again.

**Comment by jellegerbrandy** [comment]
should be resolved witht he #165
